src/Sulu/Bundle/AdminBundle/Resources/js/components/PermissionHint/README.md

```diff
@@ -3,3 +3,6 @@ This component shows a simple indicator telling the user that some permissions a
 ```javascript
 <PermissionHint />
 ```
+
+This component is deprecated and will be removed in the next major version.
```

Comment:
```suggestion
This component is deprecated and will be removed in 3.0 version.
```

When we explicitly deprecate it with 3.0, we can also use the version number here

Reply:
yeah this should be `2.6` not `3.0`